### PR TITLE
feat: add create react app example

### PR DESCRIPTION
**Pull request by twhy**
# Description

This PR
1. Removed webpack configs in `next.config.mjs` in nextjs example
2. Add a `create-react-app` example that use Lumos in browsers without extra configs.

**Why there's no Vite example ?**
I tried to create a vite example in `lumos/examples` but it/vite/rollup didn't convert commonjs modules to es6.
So when I import and use Lumos there's a `exports is not defined` error, and I didn't find any solution yet.
There's no such errors when use vite with standalone projects (not in monorepo)

We can publish a preivew version of this branch, e.g `0.30.0.preview` and create standalone projects (vite, nextjs, create-react-app) to install the preview version and test the browser compatibility related changes.

## Type of change

Please delete options that are not relevant.
- [x] Example

# How Has This Been Tested?
Unit Tests

### CODE REVIEW

File: examples/create-react-app/package-lock.json
Comment (homura) [inline examples/create-react-app/package-lock.json:1]: It should switch to the pnpm lock file to keep consistency with other examples.
You can use [`pnpm create react-app`](https://pnpm.io/cli/create) to initial the project

**Comment by homura** [comment]
I couldn't find a use case related to Lumos in this example. Could you please import Lumos to the `window.lumos` to demonstrate its compatibility with CRA without additional configurations?

File: examples/create-react-app/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@ckb-lumos/lumos": "workspace:^",
```
Comment (homura) [inline examples/create-react-app/package.json]: please switch to 'canary' to enable the example to work on the CodeSandbox

**Comment by twhy** [comment]
> I couldn't find a use case related to Lumos in this example. Could you please import Lumos to the `window.lumos` to demonstrate its compatibility with CRA without additional configurations?

Yes. I already did it in `examples/create-react-app/src/index.js`

File: examples/create-react-app/package-lock.json
Comment (twhy) [reply examples/create-react-app/package-lock.json:1]: Updated.

File: examples/create-react-app/package.json (hunk 1 above)
Comment (twhy) [reply examples/create-react-app/package.json]: Updated.

File: examples/create-react-app/package-lock.json
Comment (homura) [inline examples/create-react-app/package-lock.json:1]: npm lock can be removed now

**Comment by homura** [comment]
> There's no such errors when use vite with standalone projects (not in monorepo)

The Lumos without node modules have not been published yet. How can you confirm that Lumos works fine with standalone projects created by Vite?

File: examples/create-react-app/package-lock.json
Comment (twhy) [reply examples/create-react-app/package-lock.json:1]: Removed

**Comment by twhy** [comment]
> > There's no such errors when use vite with standalone projects (not in monorepo)
> 
> The Lumos without node modules have not been published yet. How can you confirm that Lumos works fine with standalone projects created by Vite?

1. I created a standalone vite project and install `@ckb-lumos/bi` (commonjs) and run `console.log(BI.from(1).add(99).toString())` and it works.
2. nextjs and create-react-app examples show that our latest changes works.

So to simulate a developer's behavior, we probably need to publish a preview/dev version, and setup 3 (vite, nextjs, create-react-app) standalone proejcts, install Lumos and test it to make sure everything works.

**Comment by homura** [comment]
> I created a standalone vite project and install @ckb-lumos/bi

I think `@ckb-lumos/bi` is too simple to be used as an example. Could you please validate it with [Verdaccio](https://github.com/ckb-js/lumos/blob/develop/devtools/verdaccio.yaml) in your local environment?

**Comment by twhy** [comment]
> > I created a standalone vite project and install @ckb-lumos/bi
> 
> I think `@ckb-lumos/bi` is too simple to be used as an example. Could you please validate it with [Verdaccio](https://github.com/ckb-js/lumos/blob/develop/devtools/verdaccio.yaml) in your local environment?

I try [Verdaccio](https://github.com/ckb-js/lumos/blob/develop/devtools/verdaccio.yaml) but can't add user.

So I published all packages with a different name

[`ckt-lumos-bi@0.30.0`](https://www.npmjs.com/package/ckt-lumos-bi)
[`ckt-lumos-base@0.30.0`](https://www.npmjs.com/package/ckt-lumos-base)
...
[`ckt-lumos-lumos@0.30.0`](https://www.npmjs.com/package/ckt-lumos-lumos)

and created 3 standalone projects

[`ckt-lumos-cra`](https://github.com/twhy/ckt-lumos-cra)
[`ckt-lumos-vite`](https://github.com/twhy/ckt-lumos-vite)
[`ckt-lumos-next`](https://github.com/twhy/ckt-lumos-next)

installed `ckt-lumos-lumos@0.30.0` and exposed a `lumos` object to `window.`

They all work well.

**Comment by homura** [comment]
Verified with Vite by installing `ckt-lumos-lumos`, it works well

**Review by homura — APPROVED** [review]

**Pull request merged**